mlir/include/mlir/Dialect/LLVMIR/NVVMOps.td

```diff
@@ -980,13 +980,10 @@ def NVVM_MBarrierTryWaitParityOp : NVVM_PTXBuilder_Op<"mbarrier.try_wait.parity"
   let assemblyFormat = "$addr `,` $phase `,` $ticks attr-dict `:` type(operands)";
 }
 
-def NVVM_MBarrierTestWaitOp : NVVM_Op<"mbarrier.test.wait">,
-  Results<(outs I1:$res)>,
-  Arguments<(ins AnyTypeOf<[LLVM_PointerGeneric, LLVM_PointerShared]>:$addr,
-                 I64:$state)> {
+def NVVM_MBarrierTestWaitOp : NVVM_Op<"mbarrier.test_wait"> {
```

Comment:
MLIR slightly prefers "_" on the OP name and "." is only used to separate the dialect name. We have wrong name convention in NVVM. We mix "_" and ".". 

I'd like to change all the names to "_", and it's better to do it in single PR. 

Let's not change the name here